language/getting-started/intro_vertex_ai_studio.md

```diff
@@ -0,0 +1,171 @@
+# Getting started with Vertex AI Studio
+
+| | |
+|-|-|
+|Author(s) | [Polong Lin](https://github.com/polong-lin) |
```

Comment:
This should be `[Thu Ya Kyaw](https://github.com/iamthuya)` not me actually :)

Reply:
Thanks! I think you originally committed it, so that's why you're listed as author.